awx_collection/tests/integration/targets/tower_export/tasks/main.yml

```diff
@@ -0,0 +1,77 @@
+---
+- name: Generate a random string for test
+  set_fact:
+    test_id: "{{ lookup('password', '/dev/null chars=ascii_letters length=16') }}"
+  when: test_id is not defined
+
+- name: Generate names
+  set_fact:
+    org_name1: "AWX-Collection-tests-tower_export-organization-{{ test_id }}"
+    org_name2: "AWX-Collection-tests-tower_export-organization2-{{ test_id }}"
+    inventory_name1: "AWX-Collection-tests-tower_export-inv1-{{ test_id }}"
+
+- block:
+    - name: Create some organizations
+      tower_organization:
+        name: "{{ item }}"
+      loop:
+        - "{{ org_name1 }}"
+        - "{{ org_name2 }}"
+
+    - name: Create an inventory
+      tower_inventory:
+        name: "{{ inventory_name1 }}"
+        organization: "{{ org_name1 }}"
+
+    - name: Export all tower assets
+      tower_export:
+        all: True
+      register: all_assets
+
+    - assert:
+        that:
+          - all_assets is not changed
+          - all_assets is successful
+          - all_assets['assets']['organizations'] | length() >= 2
```

Comment:
👍  I'm fine with this structure for tests. We need to confirm that they work, but this is what I expected.